GenderPayGap.WebUI/Properties/launchSettings.json

```diff
@@ -20,7 +20,7 @@
       "commandName": "Project",
       "launchBrowser": true,
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Local"

```

Comment:
I Noticed my ASP.NET version automatically defaults local builds to 'Local' instead of 'Development', as per newer standards in .NET I assume.